web/templates/nav.html

```diff
@@ -17,14 +17,14 @@
 
 	<div class="collapse navbar-collapse" id="navbarSupportedContent">
 		<ul class="navbar-nav align-items-center">
-			<li class="nav-item active ml-md-3">
-				<a class="nav-link" href="#">
-					Home<span class="sr-only">(current)</span>
+			<li class="nav-item ml-md-3">
+				<a class="nav-link" href="/">
+					Home
 				</a>
 			</li>
 			<li class="nav-item ml-md-3">
-				<a class="nav-link" target="_blank" rel="noopener"
-					href="https://endeneublog.wordpress.com/2019/10/02/artbot-takes-shape/">
+				<a class="nav-link" rel="noopener"
+					href="/about">
```

Comment:
In the future, we should make the website a SPA and just pull the body of the about (and cache) when this link is clicked.